.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Provide a general summary of the issue in the Title above -->
```

Comment:
I think it makes copy/pasting a challenge unless someone has that tribal knowledge that they're supposed to view the raw markdown.  Even with the HTML comments I recommend placing everything in a code block to ease copy/pasting.